src/main/java/com/emc/pravega/perf/PravegaWriterWorker.java

```diff
@@ -20,28 +20,26 @@
 
 import java.time.Instant;
 import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
 import io.pravega.client.stream.EventStreamWriter;
 import io.pravega.client.ClientFactory;
 import io.pravega.client.stream.Transaction;
 import io.pravega.client.stream.impl.UTF8StringSerializer;
 import io.pravega.client.stream.EventWriterConfig;
 import io.pravega.client.stream.TxnFailedException;
-import java.util.concurrent.atomic.AtomicInteger;
 
 /**
  * class for Pravega writer/producer.
  */
 public class PravegaWriterWorker extends WriterWorker {
     final EventStreamWriter<String> producer;
 
-    PravegaWriterWorker(int sensorId, int eventsPerWorker, int secondsToRun,
+    PravegaWriterWorker(int sensorId, int events, int secondsToRun,
                         boolean isRandomKey, int messageSize, Instant start,
-                        PerfStats stats, String streamName, ClientFactory factory) {
+                        PerfStats stats, String streamName, ThroughputController tput,ClientFactory factory) {
```

Comment:
Minor: space needed after `,`.

Reply:
yes , I copied the checkstyle from pravega